infrastructure-playbooks/purge-cluster.yml

```diff
@@ -287,8 +287,6 @@
      state: absent
 
   tasks:
-  - include_vars: roles/ceph-common/defaults/main.yml
-  - include_vars: roles/ceph-osd/defaults/main.yml
```

Comment:
As far as I can tell there is nothing in the "purge ceph osd cluster" play that would require anything defined in these defaults. The only value it's using that's even defined in these defaults is ``devices``.

I'm honestly not even sure why we're trying to include files from a relative ``group_vars`` directory. In my experience the correct way to use ansible is to have a ``group_vars`` directory relative to the inventory being used, not the playbook.